src/components/booking/index.tsx

```diff
@@ -101,6 +101,7 @@ export default function BookingComponent({
         booking={formData as unknown as Booking}
         onSuccess={() => router.replace("/success")}
         bookingId={id}
+        steps={["fuel", "sign", "validateFront", "validateBack", "uploadFrontIdImage", "uploadBackIdImage", "pay", "saveData", "notifyCustomer"]}
```

Comment:
You need to merge with the parent PR, I added a step after `validateBack` that I was missing this morning (sorry!)